src/plugins/orangehrmClaimPlugin/entity/Decorator/ClaimRequestDecorator.php

```diff
@@ -90,9 +91,12 @@ public function getCurrencyByCurrencyId(string $currencyId): ?CurrencyType
         return $this->getPayGradeService()->getPayGradeDao()->getCurrencyById($currencyId);
     }
 
-    public function setEmployeeByUserId(int $userId)
+    /**
+     * @param int $empNumber
+     */
+    public function setEmployeeByEmpNumber(int $empNumber)
```

Comment:
:thinking:

Reply:
to set the employee who owns the request.

Reply:
```suggestion
    public function setEmployeeByEmpNumber(int $empNumber): void
```